src/vmware/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## 3.2.0
 
 - Add `az vmware workload-network dhcp` command group
+- Add `az vmware workload-network dns-service` command group
+- Add `az vmware workload-network dns-zone` command group
+- Add `az vmware workload-network port-mirroring` command group
```

Comment:
Do you need to upgrade a new version for these changes?

Reply:
No, we have not released `3.2.0` yet.